wallet/how-to/connect/set-up-sdk/mobile/ios.md

```diff
@@ -16,12 +16,39 @@ iOS SDK GitHub repository for advanced use cases.
 
 ## Prerequisites
 
-An iOS project set up with iOS version 14+.
+- MetaMask Mobile version 7.6.0 or above installed on your target device (that is, a physical device
```

Comment:
nit: It seems like we use "or later", "or above", or "Version >number>+ in the docs. I think MS guide prefers "later".